src/calculateRentalCost.js

```diff
@@ -1,10 +1,12 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
```

Comment:
Avoid using literals in expressions. Define a constant for the daily cost of the rental.